packages/markdown-docx/src/OOXMLTransformer/index.js

```diff
@@ -15,22 +15,52 @@
 'use strict';
 
 const xmljs = require('xml-js');
-const typeOf = require('type-of');
+
 const { NS_PREFIX_CommonMarkModel } = require('@accordproject/markdown-common').CommonMarkModel;
+const { NS_PREFIX_CiceroMarkModel } = require('@accordproject/markdown-cicero').CiceroMarkModel;
 
 /**
  * Transforms OOXML to CiceroMark
  */
 class OoxmlTransformer {
     /**
-     * Gets the id of the variable
+     * Defines the JSON XML array for blocks
+     */
+    constructor() {
+        // for storing the xml tags present in a node containing its properties in array form
+        this.JSONXML = [];
+
+        // nodes of a paragraph or heading element
+        this.nodes = [];
```

Comment:
Any reason why these are instance variables? Their content would be cleared after each block element is traversed so I think it would be better to pass such elements as parameters to functions where you need them.

Reply:
I was not able to properly use them in a function thats is the solo reaon i used them as above.